### PR TITLE
PI-2097 update entity to fix pod crash

### PR DESCRIPTION
**Pull request by achimber-moj**

### CODE REVIEW

File: projects/court-case-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/entity/Entities.kt [hunk 1]
```diff
@@ -27,7 +27,7 @@ class AssessmentType(@Id val assessmentTypeId: Long, val description: String)
 class CaseAllocation(@Id val caseAllocationId: Long, val eventId: Long)
 
 @Entity
-class Court(@Id val courtId: Long, val courtName: String)
+class Court(@Column(name = "court_id") @Id val courtId: Long, val courtName: String)
```
Comment (marcus-bcl) [inline projects/court-case-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/entity/Entities.kt:30]: The dev sources aren't included in the packaged image, so I don't think this entity will be the one causing the problem in the pod.  Also the `courtId` field name would map to `court_id` for the column name anyway.

Comment (achimber-moj) [reply projects/court-case-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/entity/Entities.kt:30]: i realised this after i pushed, and i am fixing this

**Review by pmcphee77 — APPROVED** [review]

File: projects/court-case-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/generator/DocumentEntityGenerator.kt [hunk 2]
```diff
@@ -1,15 +1,12 @@
 package uk.gov.justice.digital.hmpps.data.generator
 
-import uk.gov.justice.digital.hmpps.data.entity.Court
-//import uk.gov.justice.digital.hmpps.data.entity.Institution
```
[the remark this answers is not in the document]
Comment (achimber-moj) [reply projects/court-case-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/generator/DocumentEntityGenerator.kt:4]: they have been already, see https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/feature/PI-2097-fix-entity-class/projects/court-case-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/generator/DocumentEntityGenerator.kt

[the remark this answers is not in the document]
Comment (marcus-bcl) [reply projects/court-case-and-delius/src/dev/kotlin/uk/gov/justice/digital/hmpps/data/generator/DocumentEntityGenerator.kt:4]: Sorry I see.  They were commented out in the last PR - I didn't get chance to have a look over that.

**Review by marcus-bcl — COMMENTED** [review]
Might be worth running the pipeline from the branch to make sure it fixes the pod crash before merging?

**Review by pmcphee77 — APPROVED** [review]

**Review by marcus-bcl — APPROVED** [review]

**Pull request merged**